jspwiki-main/src/main/java/org/apache/wiki/preferences/Preferences.java

```diff
@@ -125,19 +133,21 @@ public static void reloadPreferences( final PageContext pageContext ) {
      * @param request
      * @param prefs The default hashmap of preferences
      */
-	private static void parseJSONPreferences( final HttpServletRequest request, final Preferences prefs ) {
-        final String prefVal = TextUtil.urlDecodeUTF8( HttpUtil.retrieveCookieValue( request, "JSPWikiUserPrefs" ) );
-        if( prefVal != null ) {
+	private static void parseJSONPreferences( final Preferences prefs, final String prefVal ) {       
```

Comment:
it would be nice if this method is made package-private, so some unit tests defining its behaviour can be made. The alternative would be unit testing the `reloadPreferences` method, but the first approach seems easier.

Reply:
I assume this meant only changing signature to package scope. Changed method modifier to <none> in the latest commit.